spec/controllers/api/v2/accounts/report_controller_spec.rb

```diff
@@ -31,8 +32,8 @@
         super().merge(
           metric: 'conversations_count',
           type: :account,
-          since: date_timestamp.to_s,
-          until: date_timestamp.to_s
+          since: start_of_today.to_s,
+          until: end_of_today.to_s
```

Comment:
@sojan-official I had to make this change because of the range of changes we made. Earlier, when including the ends of the range, an extra day was added, which made this test work fine.

Right now, excluding the ends, make he test break, but I believe the current behaviour is correct. LMK whenever you have time to discuss this

CC: @pranavrajs @tejaswinichile